pkg/config/generate.go

```diff
@@ -379,16 +379,42 @@ func Generate(cs *api.OpenShiftManagedCluster, pluginConfig api.PluginConfig) (e
 		}
 	}
 
-	// TODO: Remove these password operations before GA
-	if len(c.AdminPasswd) == 0 {
-		if c.AdminPasswd, err = randomString(10); err != nil {
-			return err
+	// set the user passwords when testing
+	if Derived.RunningUnderTest() {
+		if len(c.DedicatedAdminPasswd) == 0 {
```

Comment:
this portion of the code is starting to look like copy/paste.  Maybe a function is in order.

Reply:
Added a function to handle these users.